seed/csharp-sdk/api-wide-base-path/src/SeedApiWideBasePath/Core/Public/ClientOptions.cs

```diff
@@ -3,7 +3,7 @@
 
 #nullable enable
 
-namespace SeedBasicAuth.Core;
+namespace SeedApiWideBasePath;
```

Comment:
example of public `Core` classes now in the root namespace

they're also in `Core/Public`, inspired by [Stripe](https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Constants/ApiVersion.cs)